### PR TITLE
Added a vox modpack.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
If I open another draft PR before my others are closed, please repoban me until I recover.

Adds vox and the main vox away site as a modpack.
**TODO**
- [X] Copy and convert all the vox content from Bay.
- [x] Split up the gear files, organize the other files into folders in the pack.
- [x] Add modpack strings for dreams etc.
- [x] Change sprite_sheets to hook into the Nebula icon system (single icon file per clothing item).
- [x] ~~Port the vox ship from Argos, possibly replace the voxbox.~~

🆑
add: Added the Vox species and associated gear from Bay as a modpack.
/🆑

### CODE REVIEW

File: mods/vox/organs_vox.dm [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+	name = "gizzard"
+	color = "#0033cc"
+	var/global/list/gains_nutriment_from_inedible_reagents = list(
+		/decl/material/solid/wood =           3,
+		/decl/material/liquid/cleaner =       1,
+		/decl/material/liquid/foaming_agent = 1,
+		/decl/material/liquid/surfactant =    1,
+		/decl/material/liquid/paint =         1
+	)
+	var/global/list/can_digest_matter = list(
```
Comment (MistakeNot4892) [inline mods/vox/organs_vox.dm]: TODO convert all these to decls. Use scav stomach as a reference.

File: mods/vox/gear/gun_spikethrower.dm [hunk 2]
```diff
@@ -38,16 +38,25 @@
...
-	icon = 'icons/obj/guns/launcher/voxspike.dmi'
-	icon_state = ICON_STATE_WORLD
+	icon = 'mods/vox/icons/spikethrower.dmi'
+	icon_state = "spikethrower3"
+	item_state = "spikethrower"
 	fire_sound_text = "a strange noise"
 	fire_sound = 'sound/weapons/spike.ogg'
 
 /obj/item/gun/launcher/alien/spikethrower/on_update_icon()
-	icon_state = "[get_world_inventory_state()][Clamp(ammo,0,3)]"
```
Comment (MistakeNot4892) [inline mods/vox/gear/gun_spikethrower.dm]: TODO undo, port correct icons from Neb folder.

File: mods/vox/datum/species.dm [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+			FACTION_VOX_CREW,
+			FACTION_VOX_APEX
+		),
+		TAG_RELIGION =  list(
+			RELIGION_VOX
+		)
+	)
+
+/*
+	exertion_effect_chance = 10
```
Comment (MistakeNot4892) [inline mods/vox/datum/species.dm]: TODO identify PRs from Bay to port to get this system.

File: mods/vox/away/voxship/voxship_radio.dm [hunk 4]
```diff
@@ -0,0 +1,31 @@
+/obj/machinery/telecomms/hub/map_preset/voxship
+	//preset_name = "Vox"
```
Comment (MistakeNot4892) [inline mods/vox/away/voxship/voxship_radio.dm]: TODO fix this file

File: mods/vox/away/voxship/voxship_jobs.dm [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+		SKILL_HAULING = SKILL_BASIC,
+		SKILL_EVA = SKILL_EXPERT,
+		SKILL_SCIENCE = SKILL_ADEPT,
+		SKILL_PILOT = SKILL_ADEPT,
+		SKILL_COMBAT = SKILL_ADEPT,
+		SKILL_WEAPONS = SKILL_ADEPT
+	)
+
+	max_skill = list(
+		SKILL_BUREAUCRACY = SKILL_MAX,
```
Comment (MistakeNot4892) [inline mods/vox/away/voxship/voxship_jobs.dm]: TODO add literacy

File: mods/vox/_vox.dme [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+#include "datum/unit_testing.dm"
+#include "gear/gear.dm"
+#include "gear/gun.dm"
+#include "gear/gun_slugsling.dm"
+#include "gear/gun_spikethrower.dm"
+#endif
+// BEGIN_INTERNALS
+// END_INTERNALS
+// BEGIN_FILE_DIR
+#define FILE_DIR .
```
Comment (MistakeNot4892) [inline mods/vox/_vox.dme]: TODO remove this crap.

File: mods/vox/away/voxship/voxship_radio.dm (hunk 4 above)
Comment (MistakeNot4892) [reply mods/vox/away/voxship/voxship_radio.dm]: Oh god this is tied to a big-ass tcomms refactor on Bay.
Pain.

**Comment by MistakeNot4892** [comment]
Decided to ditch the away site from a combination of not wanting to redo tcomms and also not being sure if submaps are worth keeping as a pattern on Neb. Just need to put species offsets on these buggers and this will be ready to go.

**Comment by MistakeNot4892** [comment]
All set.

**Review by comma — DISMISSED** [review]

**Comment by comma** [comment]
conflicts

**Review by comma — APPROVED** [review]

**Pull request merged**